ci/vm/entrypoint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -xe
```

Comment:
Better to use `base -eEx` + `set -o pipefail` + `trap ERR`

Reply:
The current Bash script is too simple for traps.
Adding flags per your recommendation.

Reply:
Without `trap ERR` it may be tricky to debug when the script exited without any output because shell command failed and `set -eE` enabled

Reply:
Traps added, thanks for the explanation.